content/posts/blog/xmpp-summit-21.md

```diff
@@ -0,0 +1,29 @@
+
+This year, the [XMPP Standards Foundation](https://xmpp.org) again gathered in force to attend the summit, that traditionally precedes the [FOSDEM event](https://fosdem.org/2017/) in Brussels, Belgium. Barely fitting in the (rather sizable) room that was made available to us by Cisco, the XSF members had a fruitful two-day meeting. 
+
+<blockquote class="twitter-tweet tw-align-center" data-lang="en"><p lang="en" dir="ltr">Day two <a href="https://twitter.com/xmpp">@xmpp</a> Summit21. <a href="https://twitter.com/hashtag/xmpp2017?src=hash">#xmpp2017</a> <a href="https://twitter.com/hashtag/IoT?src=hash">#IoT</a> <a href="https://t.co/Z0xNFBdoiY">pic.twitter.com/Z0xNFBdoiY</a></p>&mdash; Rikard Strid (@rikardstrid) <a href="https://twitter.com/rikardstrid/status/827444126962561024">February 3, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+The attending members, skillfully herded by Kev, addressed an impressive number of topics, including:
+
+* BIND2, improving the data exchange that occurs when an XMPP entity initially connects to a server.
+* [MIX](http://xmpp.org/extensions/xep-0369.html), the XEP that intends to be a replacement for the existing MUC protocol, bringing an up-to-date feature set and better extensibility.
+* Addressing the annoyance of instant-messaging-based spam.
+* The application of existing, or to be developed XMPP standards, to facilitate the growing world of [Internet-of-Things](https://wiki.xmpp.org/web/IoT_SIG).
+* The XSF intention to take part in this years [Google Summer of Code](https://summerofcode.withgoogle.com/).
+* Improvements to the existing end-to-end encryption, as defined in the [OMEMO XEP](https://xmpp.org/extensions/xep-0384.html).
+
+The full minutes are made available in the [XSF wiki](https://wiki.xmpp.org/web/Summit_21_minutes). If any of these topics (or others for that matter) interest you, we'd love to hear from you. Please find us at [xmpp.org](https://xmpp.org), use the mailinglists to contact us, or find us in one of the dedicated MUCs.
+
+<blockquote class="twitter-tweet tw-align-center" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/hashtag/XMPP2017?src=hash">#XMPP2017</a> Summit Dinner! <a href="https://t.co/cbJCztuhmn">pic.twitter.com/cbJCztuhmn</a></p>&mdash; Marcelo Terres (@mhterres) <a href="https://twitter.com/mhterres/status/827225714000797696">February 2, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Many thanks to everyone involved in making the summit happen, including [Cisco Belgium](http://www.cisco.be), [Surevine](https://www.surevine.com), [AG Software](https://www.ag-software.net), [Isode](https://www.isode.com), [Prosody](https://prosody.im), [Erlang Solutions](https://www.erlang-solutions.com/) and [Clayster](http://www.clayster.com) for providing much appreciated sponsoring for the event, as well as the traditional XSF Member dinner!
```

Comment:
Please, end all these URLs with a trailing /, as that will get rid of a redirect for each link that doesn't have it. :)  As far as I can tell, the host parts are correct now.